packages/react-events/src/__tests__/TouchHitTarget-test.internal.js

```diff
@@ -330,7 +330,7 @@ describe('TouchHitTarget', () => {
       ReactDOM.render(<Test />, container);
       expect(Scheduler).toFlushWithoutYielding();
       expect(container.innerHTML).toBe(
-        '<div><div style="position: absolute; z-index: -1; bottom: -10px; ' +
+        '<div><div style="position: absolute; z-index: -1; pointer-events: auto; bottom: -10px; ' +
```

Comment:
Can we exclude this style altogether when hydrating?

Reply:
The question is what is the least likely to cause us to invalidate the layout. This needs testing but I was hoping that the pointer events flip would be the least likely to cause massive invalidation when we hydrate it.

Reply:
I can take a look again but this was done for a specific reason at the time.

Reply:
Fixed. I realized we can set `pointer-events` to `null` and it will remove the `none` value from SSR. :)